frappe/__init__.py

```diff
@@ -448,7 +448,8 @@ def sendmail(recipients=[], sender="", subject="No Subject", message="No Message
 	:param args: Arguments for rendering the template
 	:param header: Append header in email
 	"""
-
+	if not sender:
```

Comment:
The default sender is added in `queue.py`
https://github.com/frappe/frappe/blob/1831021a77968f8770d060900849872c69108e5a/frappe/email/queue.py#L75-L77

No reason to do this here

Reply:
Hi Scmmishra,

Thanks for the reply. You are right about the change in ``frappe/__init__`` is not necessary. Howerver, we perhaps need to change the documentation (or comments) in line 438:
   ``:param sender: Email sender. Default is current user.`` to ``Default is outgoing email account``
   
As the codes you show at ``queue.py``, it fetches the outgoing_email_account's email address, not the current user's email address. For example, in my instance, I use support (support@...) as outgoing email account, but use administrator as user for set up S3-backup (admin@....).

But it should be all good, not a major problem. Last thing, I am not very sure how to  "port this fix to version-12-hotfix branch", could you please specify the steps for me?

Thank you,